app/frontend/src/components/forms/manage/AddTeamMember.vue

```diff
@@ -220,8 +220,8 @@ export default {
                 <template v-else>
                   <v-list-item
                     v-bind="props"
-                    :title="`${item?.raw?.fullName} (${item?.raw?.email})`"
-                    :subtitle="`${item?.raw?.username} (${item?.raw?.idpCode})`"
+                    :title="`${item?.fullName} (${item?.email})`"
+                    :subtitle="`${item?.username} (${item?.idpCode})`"
```

Comment:
Hmm, something isn't working here. I'm seeing "undefined" for the fields:
<img width="179" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/f1ae9ab7-2036-4891-ad09-993df3af52ad">

Reply:
another very good catch, looking into this now

Reply:
figured it out. turns out they removed "raw" from their tables when you're accessing the data but in their other components they kept it. weird decision to not keep things consistent.

Reply:
Looking good. They'll probably remove "raw" from the rest of them in the next update!